docs/aws/aws-with-windows-server.md

```diff
@@ -0,0 +1,618 @@
+# OpenShift Container Platform 4.4 on AWS with Microsoft Windows Server 2019/1809 Nodes
```

Comment:
Please add these files under  to [tools/ansible/docs](https://github.com/openshift/windows-machine-config-bootstrapper/tree/master/tools/ansible/docs).

Reply:
Sure thing; just moved them. :+1: